join.spec.md

```diff
@@ -20,20 +20,21 @@ graph LR
   style core fill:none,stroke:fuchsia,color:fuchsia;
 ```
 
-This document defines schema elements for describing [core schemas](/core/v0.1) which **join** multiple **subgraph** schemas into a single **supergraph** schema.
+This document defines a [core feature](https://specs.apollo.dev/core/v0.1) named `join` for describing [core schemas](https://specs.apollo.dev/core/v0.1) which **join** multiple **subgraph** schemas into a single **supergraph** schema.
 
 This specification provides machinery to:
 - define [subgraphs](#def-subgraph) with the {join__Graph} enum and the {@join__graph} directive
 - assign fields to subgraphs with {@join__field}
-- declare additional data required and provided by subgraph field resolvers with the [`requires`](#@join__field/requires) and [`provides`](#@join__field/provides) arguments
+- declare additional data required and provided by subgraph field resolvers with the `requires` and `provides` arguments to {@join__field}
+- assign keys and ownership to types with {@join__type} and {@join__owner}
 
 # How to read this document
 
-This document uses [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) guidance regarding normative terms: MUST / MUST NOT / REQUIRED / SHALL / SHALL NOT / SHOULD / SHOULD NOT / RECOMMENDED / MAY / OPTIONAL
+This document uses [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) guidance regarding normative terms: MUST / MUST NOT / REQUIRED / SHALL / SHALL NOT / SHOULD / SHOULD NOT / RECOMMENDED / MAY / OPTIONAL.
 
 ## What this document isn't
 
-This document specifies only the structure and semantics of supergraphs. It's expected that a supergraph will generally be the output of a compilation process which composes subgraph schemas. The mechanics of that process are not specified normatively here; a suggestion is provided in [Appendix: Basic Composition Algorithm](#sec-Appendix-Suggested-Composition-Algorithm). Conforming implementations may choose any approach they like, so long as the result conforms to the requirements of this document.
+This document specifies only the structure and semantics of supergraphs. It's expected that a supergraph will generally be the output of a compilation process which composes subgraph schemas. The mechanics of that process are not specified normatively here. Conforming implementations may choose any approach they like, so long as the result conforms to the requirements of this document.
```

Comment:
```suggestion
This document specifies only the structure and semantics of supergraphs. It's expected that a supergraph will generally be the output of a compilation process which composes subgraphs. The mechanics of that process are not specified normatively here. Conforming implementations may choose any approach they like, so long as the result conforms to the requirements of this document.
```
Not your change, but...this feels redundant, and I think we want this to collapse to subgraph in the vernacular.

Reply:
Makes sense to me, for parallelism with "supergraph" at the beginning of the sentence.